SignalServiceKit/src/Storage/TSStorageManager.h

```diff
@@ -6,6 +6,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+// TODO: Rename to OWSPrimaryStorage?
```

Comment:
WDYT?

Reply:
:+1:

Reply:
I'm going to hold off on this kind of refactoring until we merge the feature branch.